src/TranslationUi/Term/MetaboxAction.php

```diff
@@ -0,0 +1,65 @@
+<?php # -*- coding: utf-8 -*-
+declare(strict_types=1);
+
+namespace Inpsyde\MultilingualPress\YoastSeoSync\TranslationUi\Term;
+
+use Inpsyde\MultilingualPress\Framework\Admin\Metabox\Action;
+use Inpsyde\MultilingualPress\Framework\Admin\PersistentAdminNotices;
+use Inpsyde\MultilingualPress\Framework\Http\Request;
+use Inpsyde\MultilingualPress\TranslationUi\MetaboxFieldsHelper;
+use Inpsyde\MultilingualPress\TranslationUi\Term;
+
+final class MetaboxAction implements Action
+{
+    /**
+     * @var MetaboxFields
+     */
+    private $metaboxFields;
+
+    /**
+     * @var MetaboxFieldsHelper
+     */
+    private $fieldsHelper;
+
+    /**
+     * @var Term\RelationshipContext
+     */
+    private $relationshipContext;
+
+    public function __construct(
+        MetaboxFields $metaboxFields,
+        MetaboxFieldsHelper $fieldsHelper,
+        Term\RelationshipContext $relationshipContext
+    ) {
+
+        $this->metaboxFields = $metaboxFields;
+        $this->fieldsHelper = $fieldsHelper;
+        $this->relationshipContext = $relationshipContext;
+    }
+
+    public function save(Request $request, PersistentAdminNotices $notices): bool
+    {
+        $values = $request->bodyValue(
+            'multilingualpress',
+            INPUT_POST,
+            FILTER_UNSAFE_RAW,
+            FILTER_FORCE_ARRAY
+        );
+
+        $focuskw = $values["site-{$this->relationshipContext->remoteSiteId()}"]["yoast_wpseo_focuskw"];
+        $title = $values["site-{$this->relationshipContext->remoteSiteId()}"]["yoast_wpseo_title"];
+        $metadesc = $values["site-{$this->relationshipContext->remoteSiteId()}"]["yoast_wpseo_metadesc"];
+        $canonical = $values["site-{$this->relationshipContext->remoteSiteId()}"]["yoast_wpseo_canonical"];
```

Comment:
@Dinamiko What about assign the returned value of `$this->relationshipContext->remoteSiteId()` to a variable instead of calling the method multiple times?

Same for other method calls such as `$this->relationshipContext->remoteTermId()` that I see below